docs/client.md

```diff
@@ -0,0 +1,53 @@
+# pb-cli specification
+
+## Overview
+
+`pb-cli` is a command line tool which communicate with `pb-go` backend. Although user can just upload data to pastebin directly with `curl`, we still provide a tools to help user easily use our advanced feature such as burn-after-reading and private share.
+
+We guarantee everything that worked with `pb-cli`, also worked with `curl`.
+
+## Configuration
+
+You can use `-c file` to provide a config file. If you didn't provide one, it will find config file called `.pb-cli.toml` from current directory then home directory.
+
+Config file should be written in `toml` like:
```

Comment:
If possible use `yaml` instead due to dependency library for yaml is already in our repo. That will help us build the same standard for config file and reduce disk cost.

Reply:
If we use spf13/viper to implement configuration, both yaml and toml will be supported.

Changing into yaml will not reduce `pb-cli` binary size.

I suppose that final `pb-cli` binary file is about 15MB ~ 20MB with viper and cobra.